ros_ign_bridge/bin/ros_ign_bridge_generate_tests

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import sys
+
+# add ros_ign_bridge to the Python path
+ros_ign_bridge_root = os.path.join(os.path.dirname(__file__), '..')
+sys.path.insert(0, os.path.abspath(ros_ign_bridge_root))
+
+from ros_ign_bridge import generate_test_cpp
+
+
+def main(argv=sys.argv[1:]):
+    parser = argparse.ArgumentParser(
+        description='Generate C++ template specializations.',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument(
+        '--output-path',
+        required=True,
+        help='The basepath of the generated C++ files')
+    parser.add_argument(
+        '--template-dir',
+        required=True,
+        help='The location of the template files')
+    parser.add_argument(
+        '--gz-msgs-ver',
+        required=False,
+        default="0.0.0",
```

Comment:
can we set this value from the CMAke ? or is it optional ?

Reply:
It's being passed in as an argument from CMake: https://github.com/ignitionrobotics/ros_ign/pull/234/files#diff-1c1942d8b5067a8d162d921c4b5bdd95f38e460faf6d9bfdf2e38946c34f1abfR214

This way we don't have to do any sort of generation.